### DISCUSSION TITLE
How do you get how many cells in every clusting?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#798] [Q&A]
I notice your publication can calculate every clustering include  how many cells like below picture, how do you get it?
![baa4aa77972a40c8515133c4c80a880](https://user-images.githubusercontent.com/104721358/218736585-e049d095-f95e-4126-aa68-a58900de7a8e.png)

**Comment by Thomas-Ulrich** [comment]
that the clustering variable of the volume output.

**Reply by smallriver666** [reply]
> that the clustering variable of the volume output.
where can I see the number of cells in every clustering.
![image](https://user-images.githubusercontent.com/104721358/219028448-0dd6122c-6c53-48af-9946-38ff0ba75f8b.png)

**Reply by Thomas-Ulrich** [reply]
paraview filter histogram?

**Reply by smallriver666** [reply]
> paraview filter histogram?

Yes, I can see, but my example just has 9millions cells,why in the histogram the number much larger than 9 million.
![20230216005529](https://user-images.githubusercontent.com/104721358/219101503-2cc85606-34a9-41b0-8607-557df9d1a43c.png)

![image](https://user-images.githubusercontent.com/104721358/219101434-8b144685-998e-4c11-9506-8ef3f6d8cdaa.png)

**Reply by buzhiqiuyuebai** [reply]
> > paraview filter histogram?
> 
> Yes, I can see, but my example just has 9millions cells,why in the histogram the number much larger than 9 million. ![20230216005529](https://user-images.githubusercontent.com/104721358/219101503-2cc85606-34a9-41b0-8607-557df9d1a43c.png)
> 
> ![image](https://user-images.githubusercontent.com/104721358/219101434-8b144685-998e-4c11-9506-8ef3f6d8cdaa.png)

I think ii is caused by refimement=1, results in a tripling of the number of meshes.

**Reply by krenzland** [reply]
Yes, exactly. If you update to the most recent version of SeisSol, you can have easier access to the clustering data:
It automatically writes a [output-prefix]-clustering.csv file which should contain all information that you require.

**Reply by smallriver666** [reply]
> Yes, exactly. If you update to the most recent version of SeisSol, you can have easier access to the clustering data: It automatically writes a [output-prefix]-clustering.csv file which should contain all information that you require.

Thanks!
